src/components/DropDown.tsx

```diff
@@ -1,33 +1,45 @@
-import { useEffect, useState } from "react";
+import React from "react";
 import {
   DropDownProps,
   DropDownVarient,
   DropDownOption,
 } from "../types/DropDown";
 
-const DropDown: React.FC<DropDownProps> = ({ value, varient, onChange }) => {
-  const [options, setOptions] = useState<DropDownOption[]>([]);
+interface DropDownComponentProps extends DropDownProps {
+  data: {
+    title: string;
+    subAssets: DropDownOption[];
```

Comment:
Please rename `subAssets` because you are using this dropdown component not only for subassets